gettsim/demographic_vars.py

```diff
@@ -327,3 +327,81 @@ def rentner_in_hh(hh_id: IntSeries, rentner: BoolSeries) -> BoolSeries:
     BoolSeries indicating pensioner in household.
     """
     return rentner.groupby(hh_id).any()
+
+
+def steuerklasse(
+    tu_id: IntSeries,
+    gemeinsam_veranlagt: BoolSeries,
+    alleinerziehend: BoolSeries,
+    bruttolohn_m: FloatSeries,
+    kind: BoolSeries,
+    eink_st_params: dict,
+    eink_st_abzuege_params: dict,
+) -> IntSeries:
+    """ Determine Lohnsteuerklassen (also called 'tax brackets')
+    They determine the basic allowance for the withdrawal tax
+
+    1: Single
+    2: Single Parent
+    3: One spouse in married couple who receives allowance of both partners.
+       Only in case of Single-Earner Households
+    4: Both spouses receive their individual allowance
+    5: Other spouse in couple who receives no allowance
+    6: Additional Job...not modelled yet, as we do not
+    distinguish between different jobs
+
+    Parameters
+    ----------
+    tu_id: Tax Unit Identifier
+    gemeinsam_veranlagt: BoolSeries
+        whether or not two household members are married and file their
+        tax claim jointly
+    alleinerziehend: BoolSeries
+        Single Parent Indicator
+    bruttolohn_m: FloatSeries
+        Monthly Earnings
+    kind: BoolSeries
+        Dependent Child indicator
+    eink_st_params:
+    eink_st_abzuege_params:
+
+    Returns
+    ----------
+    steuerklasse: IntSeries
+        The steuerklasse for each person in the tax unit
+    """
+
+    bruttolohn_max = bruttolohn_m.groupby(tu_id).max()
+    bruttolohn_min = bruttolohn_m.groupby(tu_id).min()
+    erwachsener = ~kind
+    anz_erwachsene_tu = erwachsener.groupby(tu_id).transform("sum")
```

Comment:
the function `anz_erwachsene_tu` exists already and should be used as argument of this function.

Reply:
Or even better: just use `gemeinsam_veranlagt` which is just defined as `tu_id.replace(anz_erwachsene_tu) == 2`

Reply:
done